### DISCUSSION TITLE
Battle system, current proposed data model needs changes

### DISCUSSION THREAD

**Discussion by brianhhuynh38** [brianhhuynh38/slackpoint-v3#18] [Ideas]
Regarding the battle model in the GitHub wiki, I think we'll probably need to add a `turn_status` bit in there too. This would allow for the system to determine whose turn it is within the battle. Unless you've updated it to have that already? @shinegami-2002

**Comment by shinegami-2002** [comment]
I think adding a turn_status bit is a great idea. It would help us keep track of whose turn it is and make the battle system more organised, the Github wiki is now updated.
